app/views/project_information/show/_information_list.erb

```diff
@@ -35,6 +35,10 @@
       row.key { t('project_information.show.project_details.rows.advisory_board_date') }
       row.value { @project.advisory_board_date }
     end
+    summary_list.row do |row|
+      row.key { t('project_information.show.project_details.rows.sharepoint_link') }
+      row.value { link_to(@project.sharepoint_link, @project.sharepoint_link, target: :_blank) }
```

Comment:
How do you feel about adding `sanatize` to this? I think it would start to document how we are doing all we can to prevent an injection attack? I am no sure it actually covers everything, perhaps we can even limit the tags down to `href` and `target` and wrap that up in a method?

Reply:
Also, given how long, ugly and unhelpful a SharePoint url is, I think it might be better to show so words instead of the url, something like:

> Open the project folder on SharePoint in a new tab